src/main/resources/application.yml

```diff
@@ -58,6 +58,7 @@ oauth:
   issuers:
 
 storm:
+  nginx-reverse-proxy: ${STORM_WEBDAV_NGINX_REVERSE_PROXY:false}
```

Comment:
As commented for the variable in the code, I would call this parameter `behind-nginx`, whose meaning is more explanatory of what it does.